lib/task_listener.js

```diff
@@ -39,6 +30,12 @@ export default class TaskListener extends EventEmitter {
     this.runningTasks = [];
     this.taskQueue = new TaskQueue(this.runtime);
     this.taskPollInterval = this.runtime.taskQueue.pollInterval;
+
+    if (this.runtime.isolatedContainers) {
```

Comment:
Curious, can't we just remove the `isolatedContainers` option and activate it with `deviceManagement.cpu.enabled = true` ?

I might not be up to date with how config is loaded from user-data, so if `deviceManagement` can't fit in user-data forget it :)
Just and idea...